conans/test/integration/toolchains/cmake/cmakedeps/test_cmakedeps.py

```diff
@@ -62,7 +63,7 @@ class Pkg(ConanFile):
         """)
     client.save({"conanfile.py": consumer})
     client.run("install . -s:b os=Windows -s:h os=Linux -s:h compiler=gcc -s:h compiler.version=7 "
-               "-s:h compiler.libcxx=libstdc++11 --build=missing")
+               "-s:h compiler.libcxx=libstdc++11 -s:h arch=x86_64 --build=missing")
```

Comment:
Here I figured that since we are being explicit about the other settings in the command line, we can also pass the architecture to match the `client.load()` call one line below. I'm not sure there's much to gain by letting it fallback to the default profile in this particular instance